### DISCUSSION TITLE
TTY Login: some keycombos do not work in editor 'micro'

### DISCUSSION THREAD

**Discussion by developer1105** [ohmyzsh/ohmyzsh#10960] [Help]
When logging via Ctrl+Alt+F1, and opening the text editor micro, some key combinations do not work. Ctrl+Home and Ctrl+End do not work at all. 

Ctrl+Q and Ctrl+S work only if I press the Control key on the left side, if I press the Control key on the right side it does not work. 

Environment variables: TERM=linux, SHELL=zsh

**Comment by rwmitchell** [comment]
How did you determine this was an OMZ issue and not micro ?

Does the issue happen without running any startup scripts by starting zsh with ```zsh -f``` ?
Does the issue happen with other shells, such as bash?

Did you mean "When logging IN via ..." ? Or do those keystrokes on your system enable some method to log screen io to a file?

**Reply by developer1105** [reply]
I have edited the keyboard map in "/usr/share/kbd/keymaps/i386/qwerty" (to add a second control key) and now every key combination works fine in nano. (Ctrl+Home, Ctrl+End still do not work in micro. So I guess it is an specific issue related to micro, when logging via console - no X11 and no Wayland).

You are right, probably there is a better place to report this issue. I will be glad if you can point it to me.
